src/components/UserInfo.js

```diff
@@ -1,38 +1,58 @@
-/**
- * "The UserInfo class is responsible for rendering information about the user on the page."
- */
 export default class UserInfo{
   #nameElement;
   #descriptionElement;
+  #avatarElement;
   #name;
   #description;
+  #avatar;
+
+/**
+ * Manage getting and setting information in profile section of page
+ * @param {object} elements {nameElement, descriptionElement, avatarElement} elements containing relevant profile information
+ * @param {object} userInfo {name, about, avatar} strings of user information to popuplate profile information on page
+ */
+  constructor(elements, userInfo){
+    //elements for name, description, and avatar
+    this.#nameElement = elements.nameElement;
+    this.#descriptionElement = elements.descriptionElement;
+    this.#avatarElement = elements.avatarElement;
+
+    //store name, description, and avatar in object
+    this.#name = userInfo.name;
+    this.#description = userInfo.about;
+    this.#avatar = userInfo.avatar;
+
+    //display name, description, and avatar in page
+    this.#nameElement.textContent = this.#name;
+    this.#descriptionElement.textContent = this.#description;
+    this.#avatarElement.src = this.#avatar;
+  }
 
   /**
-   * Manage getting and setting information in profile section of page
-   * @param {element} name element for name
-   * @param {element} description element for description
+   * Gets name and description that are currently on page
+   * @returns {object} {username, description}
    */
-  constructor(name, description){
-    this.#nameElement = name;
-    this.#descriptionElement = description;
-    this.#name = this.#nameElement.textContent;
-    this.#description = this.#descriptionElement.textContent
+  getUserInfo(){
+    return {username: this.#name, description: this.#description};
```

Comment:
Needs correction: You should return the element contents here, for example `username:  this.#nameElement.textContent`